student-store-ui/src/components/NotFound/NotFound.css

```diff
@@ -0,0 +1,12 @@
+.not-found {
+    margin-left: 10%;
+    padding: 5% 10%;
+}
+
+.not-found h2 {
+    font-size: 100px;
```

Comment:
To create a more consistent look throughout your app, you might consider adding a default stylesheet and then adjust things from there (assuming you're not using a UI library). This not only makes the UI more polished, but can make it easier for you the developer to write predictable css. 

For me, it's helpful to override some default browser styling off the bat (ex. certain margins/padding). Something like https://necolas.github.io/normalize.css/ could be good to add to your project from the beginning.